packages/0xsequence/tests/browser/wallet-provider/dapp2.test.ts

```diff
@@ -64,23 +60,8 @@ export const tests = async () => {
     const networks = await wallet.getNetworks()
     console.log('=> networks', networks)
 
-    // There is exactly one default chain
-    assert.equal(networks.filter(network => network.isDefaultChain).length, 1, 'there is exactly one default chain')
-
-    // There's exactly one auth chain
-    assert.equal(networks.filter(network => network.isAuthChain).length, 1, 'there is exactly one auth chain')
-
-    // The default chain's chain ID is 31338
-    assert.equal(networks.filter(network => network.isDefaultChain)[0].chainId, 31338, 'default chain id is 31338')
-
-    // The non-default chain's chain ID is 31337
-    assert.equal(networks.filter(network => !network.isDefaultChain)[0].chainId, 31337, 'non-default chain id is 31337')
-
-    // The auth chain's chain ID is 31338
-    assert.equal(networks.filter(network => network.isAuthChain)[0].chainId, 31338, 'auth chain id is 31338')
-
-    // The non-auth chain's chain ID is 31337
-    assert.equal(networks.filter(network => !network.isAuthChain)[0].chainId, 31337, 'non-auth chain id is 31337')
+    assert.true(networks[0].isDefaultChain, 'network0 is defaultChain')
+    assert.true(!networks[1].isDefaultChain, 'network1 is not defaultChain')
```

Comment:
Is there a reason the test now assumes the first is the default? Isn't it more appropriate to just check that there is exactly one default chain?